pyproject.toml

```diff
@@ -0,0 +1,6 @@
+[tool.ruff]
+exclude = ["docs"]
```

Comment:
why exclude docs? I'd be up for including this.

Reply:
Sure, done.